lib/libmei4.plg

```diff
@@ -1814,14 +1819,15 @@ SetAttributes "(element, new_attrs) {
 GetId "(element) {
         return element._id;
 }"
-SetId "(element, value) {
-        olddict = Self._property:MEIFlattened;
-        oldid = element._id;
-
-        newdict = removeKeyFromDictionary(olddict, oldid);
-        element._id = value;
-        newdict[value] = element;
-        Self._property:MEIFlattened = newdict;
+SetId "(element, newId) {
+    UnregisterId(element._id);
```

Comment:
Isn’t this supposed to be `newId`? Where does `value` come from?

Reply:
Sorry — next line. Mobile phone code reviews :-/

Reply:
That's right. I made sure I unloaded/reloaded the proper files in Sibelius and ran the tests successfully.